src/app/component/user/components/profile/profile.component.html

```diff
@@ -1,27 +1,30 @@
 <section class="profile-wrapper">
-    <div class="column left-column">
-      <div class="app-profile-header">
-        <app-profile-header></app-profile-header>
-      </div>
-      <div class="app-achievements-friends">
-        <app-achievements-friends></app-achievements-friends>
-      </div>
-      <div class="app-eco-places">
-        <app-eco-places></app-eco-places>
-      </div>
-      <div class="app-shopping-list">
-        <app-shopping-list></app-shopping-list>
-      </div>
+  <div class="column left-column">
+    <div class="app-profile-header">
+      <app-profile-header></app-profile-header>
     </div>
-    <div class="column habits-dashboard">
-      <app-profile-dashboard></app-profile-dashboard>
+    <div class="app-achievements-friends">
+      <app-achievements-friends></app-achievements-friends>
     </div>
-    <div class="column right-column">
-      <div class="app-calendar">
-        <app-calendar></app-calendar>
-      </div>
-      <div class="app-profile-cards">
-        <app-profile-cards></app-profile-cards>
-      </div>
+    <div class="app-eco-places">
+      <app-eco-places></app-eco-places>
     </div>
+    <div class="app-shopping-list">
+      <app-shopping-list></app-shopping-list>
+    </div>
+  </div>
+  <div class="column habits-dashboard">
+    <app-profile-dashboard></app-profile-dashboard>
+  </div>
+  <div class="column right-column">
+    <div class="app-calendar">
+      <app-calendar></app-calendar>
+    </div>
+    <div class="app-calendar-week">
+      <app-calendar-week></app-calendar-week>
+    </div>
+    <div class="app-profile-cards">
+      <app-profile-cards></app-profile-cards>
+    </div>
+  </div>
```

Comment:
Looks like it’s mostly not your code, but rewrite without unnecessary `div` elements. E.g. we have `div` wraps one element:
```html
<div class="app-profile-cards">
   <app-profile-cards></app-profile-cards>
</div>
```
it can be rewritten more simplify:
```html
<app-profile-cards class="app-profile-cards"></app-profile-cards>
```
 The reason: the more `dom` elemets, the 'heavier' and worse it is.

Reply:
Fixed